src/main/java/com/example/school/model/Student.java

```diff
@@ -0,0 +1,7 @@
+package com.example.school.model;
+
+public class Student {
+    String name;
+    String surname;
+    Grade grade;
```

Comment:
Remove the field, will add class `Grade` in next PR.

Reply:
Also add the field(time when student start studying in the school).

Reply:
Done